db/query.go

```diff
@@ -210,6 +210,20 @@ func (context *DatabaseContext) N1QLQueryWithStats(queryName string, statement s
 	return results, err
 }
 
+// N1QlExplain issues an EXPLAIN query for the given statement
+func (context *DatabaseContext) N1QLExplain(statement string, params interface{}, consistency gocb.ConsistencyMode) (results gocb.QueryResults, err error) {
```

Comment:
It doesn't look like this is used anywhere. Was this added for completion, or an old bit of code?

Reply:
It made more sense to push this down to bucket, and I forgot to clean this up.  Pushed up a commit to remove.